DataPlotly/core/plot_settings.py

```diff
@@ -176,6 +176,7 @@ def __init__(self, plot_type: str = 'scatter', properties: dict = None, layout:
         self.data_defined_y_min = None
         self.data_defined_y_max = None
         self.source_layer_id = source_layer_id
+        self.source_layer_name = source_layer_name
```

Comment:
instead of storing this here, you're better off to retrieve it when required (i.e. in layout_item_gui.py) -- otherwise it won't respond nicely to layer renames